amundsen_application/static/js/components/ProfilePage/index.tsx

```diff
@@ -137,51 +152,56 @@ export class ProfilePage extends React.Component<ProfilePageProps> {
                       <h1>{ user.display_name }</h1>
                       {
                         (!user.is_active) &&
-                        <Flag caseType="sentenceCase" labelStyle="label-danger" text="Alumni"/>
+                        <Flag caseType="sentenceCase" labelStyle="danger" text="Alumni"/>
                       }
                     </div>
                     {
                       user.role_name && user.team_name &&
-                      <label id="user-role">{ `${user.role_name} on ${user.team_name}` }</label>
+                      <div id="user-role" className="body-2">{ `${user.role_name} on ${user.team_name}` }</div>
```

Comment:
There's probably a better tag for text, but I'm not sure it's `<label>`. Generally labels are used to describe form inputs (radio, input text, checkbox). Additionally these have some default styles that we don't want to inherit `font-weight: bold; display: inline-block; margin-bottom: 5px;`


By default I tend to use `<span>` or `<div>` since they are generally pretty neutral and don't come with too many default styles that need to be overwritten.